apps/nova/src/lib/ratings.ts

```diff
@@ -0,0 +1,57 @@
+import { IRating } from "@litespace/types";
+
+export const ratingOrganizer = (
+  ratings: IRating.FindTutorRatingsApiResponse["list"],
+  currentUserId: number
+) => {
+  let currentUserRating: IRating.RateeRatings | undefined;
+  const ratingsWithFeedback: IRating.RateeRatings[] = [];
+  const ratingsWithoutFeedback: {
+    ratings: Array<{
+      name: string | null;
+
+      userId: number;
+
+      imageUrl: string | null;
+    }>;
+    value: number;
```

Comment:
This type is the same as the `TutorRatingCardGroup` props. Please use this type and don't ceate a new one.